tests/roles/ceph_migrate/tasks/fail_mgr.yaml

```diff
@@ -0,0 +1,38 @@
+# Get a client using -v /home/tripleo-admin/ceph_config:/etc/ceph:z as input
+- name: Refresh ceph_cli
+  ansible.builtin.include_tasks: ceph_cli.yaml
+  vars:
+    ceph_config_home: "{{ ceph_config_tmp_client_home }}"
+    ceph_fsid: "{{ mon_dump.fsid }}"
+    ceph_cluster: ceph
+
+- name: Force fail ceph mgr
+  become: true
+  ansible.builtin.command: "{{ ceph_cli }} mgr fail"
+
+- name: Wait for cephadm to reconcile
+  ansible.builtin.pause:
+    seconds: "{{ ceph_wait_mon_timeout }}"
+
+- name: Get the ceph orchestrator status with
+  become: true
+  ansible.builtin.command: "{{ tripleo_cephadm_ceph_cli }} orch status --format json"
```

Comment:
as per  https://github.com/openstack-k8s-operators/data-plane-adoption/blob/main/tests/roles/ceph_migrate/tasks/ceph_cli.yaml#L19

tripleo_cephadm_ceph_cli should be changed to 'ceph_cli'

Reply:
done